src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    Animal.alive.push(this);
+  }
+
+  health = 100;
```

Comment:
`health` property should be inside the constructor same as `name`

Reply:
ok, I'll fix it, but...
![image](https://github.com/mate-academy/js_herbivores_and_carnivores/assets/104383450/b451caeb-1c86-43a7-8ff5-2e5009d9fcb1)